src/style.css

```diff
@@ -0,0 +1,19 @@
+#element_1 {
+  background: red;
+}
+
+#element_2 {
+  background: green;
+}
+
+#element_3 {
+  background: blue;
+}
```

Comment:
IDs are used for JS, try not to style the id if you can avoid it. This makes impossible to reuse the code. Stylize classes. [Style guide](https://mate-academy.github.io/style-guides/htmlcss.html#id-selectors)